extension/src/experiments/studio.ts

```diff
@@ -40,12 +47,16 @@ export class Studio extends Disposable {
     return `${this.baseUrl}?showOnlySelected=1&experimentReferences=${sha}&activeExperimentReferences=${sha}%3Aprimary`
   }
 
-  private async getBaseUrl(studioToken: string, gitRemoteUrl: string) {
+  private async getBaseUrl(
+    studioToken: string,
+    gitRemoteUrl: string,
+    remoteExpRefs: string[] | [null]
+  ) {
     try {
       const response = await fetch(`${STUDIO_URL}/webhook/dvc`, {
         body: JSON.stringify({
           client: 'vscode',
-          refs: [null],
+          refs: remoteExpRefs,
```

Comment:
@mattseddon, dvc uses `refs` as an object that supports `pushed` and `removed` keys, which are `string[]`.

See https://github.com/iterative/studio/issues/5095#issuecomment-1489777935.

Reply:
are `pushed`/`removed` optional?

Reply:
yes, both are optional, but any one of it is required to pass the `not refs` checks.

So, you could just pass `pushed: []` instead of `[null]` that would work today until you have a new API.